rpc/service/src/service.rs

```diff
@@ -977,3 +1070,12 @@ impl AsyncService for RpcCoreService {
         })
     }
 }
+
+#[derive(Debug, Default)]
+struct EstimatedFeeCache {
+    min: portable_atomic::AtomicF64,
```

Comment:
Can you explain the reasoning behind using multiple float atomics rather than an atomic arc holding the final response object? (also - adding another lib dep is another negative point for using atomic floats)

Reply:
> Can you explain the reasoning behind using multiple float atomics rather than an atomic arc holding the final response object? (also - adding another lib dep is another negative point for using atomic floats)

1) arc will allocate it on heap
2.1) that's not a new dep in codebase, it's new to rpc crate
2.2) ofc I can make it u64 and convert it to/from float by `from_bits`, `to_bits`. but what is the reason to have boilerplate code like that in our codebase when `portable-atomic` already has it

Reply:
I don't think such an heap allocation (very small, once per second) is a real consideration. And imho it will simplify the code